operations/availabilityblocks.md

```diff
@@ -196,7 +197,8 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `UpdatedUtc` | string | required | Last update date and time of the block in UTC timezone in ISO 8601 format. |
 | `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first time unit, in UTC timezone ISO 8601 format. See [Time units](../concepts/time-units.md). |
 | `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last time unit, in UTC timezone ISO 8601 format. See [Time units](../concepts/time-units.md). |
-| `ReleasedUtc` | string | required | The moment when the block and its availability is released in UTC timezone in ISO 8601 format. |
+| `ReleasedUtc` | string | optional | The moment when the block and its availability is released in UTC timezone in ISO 8601 format. Mutually exclusive with `RollingReleaseOffset`; the block will not be automaticaly released if neither `ReleasedUtc` nor `RollingReleaseOffsetUtc` is specified. |
+| `RollingReleaseOffset` | string | optional | Exact offset from the start of availability adjustments to the moment the availability adjustment should be released, in ISO 8601 duration format. Mutually exclusive with `ReleasedUtc`; the block will not be automaticaly released if neither `ReleasedUtc` nor `RollingReleaseOffsetUtc` is specified. |
```

Comment:
"not specified": does this mean that they are not sent, or does it mean they contain the value "null"?
It also states RollingReleaseOffset AND RollingReleaseOffsetUtc.
Going from not required to optional is a breaking change.

Reply:
I would interpret the two cases exactly the same, setting a property to "null" is the same as not including it, as "null" is not a value (@jnv correct me if I'm wrong). I've also raised the other two points myself, but it's good to hear it directly from an API user, thank you @remywrevcontrol. @jvn please consider these points, especially about the breaking change.

Reply:
`optional` means "can be null".

We are aware this is a breaking change and our partner success manager is already in touch with you regarding this unfortunate change.

From documentation perspective, this is a breaking change. From OpenAPI Spec perspective, the property was never marked as required in the first place which is why this wasn't flagged as a breaking change on our watch. But it really started to be null only once the domain team introduced rolling releases earlier this year.

This is a f*up on our part and we'll have a long discussion with the team responsible, unfortunately we can't do much else now other than to fix the documentation and help you to accommodate for these changes.